src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -362,54 +398,93 @@ const LostAndFound = () => {
 
       {titleCard()}
 
-      {/* Active Missing Item Reports */}
-      <Grid container justifyContent="center" spacing={3} marginTop={3}>
-        <Grid item xs={12} md={10}>
-          <Card>
-            <CardHeader
-              className="gc360_header"
-              title={
-                <Typography variant="h5" align="center">
-                  My Active <span className={styles.yellowText}>Lost</span> Item Reports
-                </Typography>
-              }
-            />
-          </Card>
+      {/* Recently Found Reports */}
+      {foundReports.length > 0 && (
+        <Grid container justifyContent="center" spacing={3} marginTop={3}>
+          <Grid item xs={12} md={10}>
+            <Card>
+              <CardHeader
+                className="gc360_header"
+                title={
+                  <Grid container alignItems="center" position="relative" justifyContent="center">
+                    {/* Badge positioned on the top-left corner */}
+                    <Badge
+                      badgeContent={foundReports.length}
+                      color="error"
+                      className={styles['badge-position']}
```

Comment:
Can we write this the way all the other css classnames are called, styles.badge-position.  I know this works, but unless there's a reason to be different here, we should do it consistently.

Reply:
That also means you should name the class badgePosition, again consistency thing.